Update the README for all projects (#195)
FuelLabs/sway-applications: fundraiser/README.md

## Overview

> A fundraiser, or crowdfund, is an application where any number of users are able to pledge towards a goal specified by the creator of the campaign. An example of a campaign could be to raise $500,000 in order to rebuild a local school park. If the target amount is reached, or surpassed, then after the deadline of the campaign the creator is able to take those funds and spend it towards the proposed goal. On the other hand, if the target is not reached then all the users that have pledged are able to withdraw their pledge.

Reviewer: Maybe avoid mentioning money.

```suggestion
A fundraiser, or crowdfund, is an application where any number of users are able to pledge towards a goal specified by the creator of the campaign. If the target amount is reached, or surpassed, then after the deadline of the campaign the creator is able to take those funds and spend it towards the proposed goal. On the other hand, if the target is not reached then all the users that have pledged are able to withdraw their pledge.
```

> A fundraiser, or crowdfund, is an application where any number of users are able to pledge towards a goal specified by the creator of the campaign. An example of a campaign could be to raise $500,000 in order to rebuild a local school park. If the target amount is reached, or surpassed, then after the deadline of the campaign the creator is able to take those funds and hopefully spend it in the way that they have claimed that they would. On the other hand if the target is not reached then all the users that have pledged are able to withdraw their pledge.

Reviewer 2: ```suggestion
A fundraiser, or crowdfund, is an application where any number of users are able to pledge towards a goal specified by the creator of the campaign. An example of a campaign could be to raise $500,000 in order to rebuild a local school park. If the target amount is reached, or surpassed, then after the deadline of the campaign the creator is able to take those funds and spend it towards the proposed goal. On the other hand, if the target is not reached then all the users that have pledged are able to withdraw their pledge.
```
